tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"globalize/date",
+	"lib/helper"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
I'm missing context I guess. Do you mean creating the globalize instance below and returning the object of formatters generated by it?